### PR TITLE
fix: autoExtendRange option and logs for simulation with signals are out of range.

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

**Comment by lpatiny** [comment]
Ok for me the PR works correctly. What you could still do maybe is more precise to tell that proton was not predicted because there are none or because there is only labile proton.

File: package-lock.json
Comment (targos) [inline package-lock.json:1]: Please cancel all changes to this file.

Comment (jobo322) [reply package-lock.json:1]: is it correct now @targos ?

Comment (targos) [reply package-lock.json:1]: yes

**Pull request merged**